releasenotes/notes/asm-blocking-fixes-fce3bb6a81a2d186.yaml

```diff
@@ -0,0 +1,9 @@
+---
+fixes:
+  - |
+    Solve some corner cases where a Flask blocking request would fail because 
+    headers would be already sent.
+  - |
+    Solve the content-type not always being correct in blocking responses.
+  - |
+    Ensure the blocking responses have some span tags.
```

Comment:
Could we possibly be a bit more specific here?

Reply:
Yes, will be on next commit.